### PR TITLE
fix: make `as_polars_series` working correctly

### PR DESCRIPTION
**Pull request by eitsupi**
Part of #570

### CODE REVIEW

**Comment by eitsupi** [comment]
I am unable to update this for the next few days due to a business trip, so someone else can take over.

File: R/as_polars.R
Comment (eitsupi) [inline R/as_polars.R:1]: TODO: `as_polars_series.RPolarsSeries` etc.

**Review by etiennebacher — APPROVED** [review]
LGTM, thanks!

**Comment by sorhawell** [comment]
Nice work. This looks very promising. I will just make a fast speed reression test on conversion nested R lists where as_polars_series could be called a large number of times.

**Review by sorhawell — APPROVED** [review]

**Comment by eitsupi** [comment]
Thanks for checking this!

**Pull request merged**